editoast/src/views/infra/objects.rs

```diff
@@ -155,6 +157,68 @@ mod tests {
         assert_eq!(call_service(&app, req).await.status(), StatusCode::OK);
     }
 
+    #[rstest]
+    async fn get_objects_should_return_track_section(#[future] small_infra: TestFixture<Infra>) {
+        let small_infra = small_infra.await;
+        let app = create_test_service().await;
+
+        let expected_track_section_object = vec![ObjectQueryable {
+            obj_id: "TA0".into(),
+            railjson: json!({
+                "curves": [],
+                "extensions": {
+                    "sncf": {
+                        "line_code": 424242,
+                        "line_name": "West_parking",
+                        "track_name": "V1",
+                        "track_number": 1
+                    }
+                },
+                "geo": {
+                    "coordinates": [
+                        [
+                            -0.4,
+                            49.5
+                        ],
+                        [
+                            -0.365,
+                            49.5
+                        ]
+                    ],
+                    "type": "LineString"
+                },
+                "id": "TA0",
+                "length": 2000.0,
+                "loading_gauge_limits": [],
+                "sch": {
+                    "coordinates": [
+                        [
+                            -0.4,
+                            49.5
+                        ],
+                        [
+                            -0.365,
+                            49.5
+                        ]
+                    ],
+                    "type": "LineString"
+                },
+                "slopes": []
+            }),
```

Comment:
You could avoid this with the creation of your own track on an empty infra (avoiding the creation of a small infra too)

Reply:
Done.